pynuodb/encodedsession.py

```diff
@@ -749,9 +767,13 @@ def getValue(self):
             return self.getBoolean()
         
         # get uuid type
-        elif typeCode in [protocol.UUID, protocol.SCALEDCOUNT1, protocol.SCALEDCOUNT2]:
+        elif typeCode is protocol.UUID:
             return self.getUUID()
-        
+
```

Comment:
I thought we agreed that protocol.SCALEDCOUNT1 would still do a pass as it did before?